src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -42,10 +41,12 @@ public AddCommand parse(String args) throws ParseException {
         Name name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
         Phone phone = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get());
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
-        Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
-        Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
+        Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_ALIAS));
```

Comment:
A suggestion: rename the variable and function name to aliasList and parseAliases accordingly if you're confident that there wont be any further changes to naming in the future